src/components/app.tsx

```diff
@@ -0,0 +1,13 @@
+import Main from '../pages/main/main';
+
+type AppProps = {
+  dataMain: {
+     RentalOffers: number;
```

Comment:
Свойства у объектов пишутся с маленькой буквы. И во множественном числе только массивы.
Критерии [Б10](https://up.htmlacademy.ru/react-individual/3/criteries#b10) и [Б5](https://up.htmlacademy.ru/react-individual/3/criteries#b5)

```suggestion
     rentalOffersAmount: number;
```

Reply:
Исправлю